collector.go

```diff
@@ -25,23 +26,23 @@ func NewErrorCollector() ErrorCollector {
 }
 
 // Report adds an error to map of aggregated errors
```

Comment:
Can we update the docs with the parameters?
or is that not something we are doing in `periskop` ?

Reply:
Which docs? It's already explained in the README

Reply:
The line were I wrote it, the internal documentation.

Just wondering if we do not provide parameter information in the internal documentation of each method

Reply:
Not really, I never saw that in any Go documentation :shrug: Fo example, prometheus client documentation: https://godoc.org/github.com/prometheus/client_golang/prometheus

Reply:
But prometheus does provide some documentation about parameters; https://godoc.org/github.com/prometheus/client_golang/prometheus#ExponentialBuckets

```golang
func ExponentialBuckets(start, factor float64, count int) []float64
```

>ExponentialBuckets creates **'count'** buckets, where the lowest bucket has an upper bound of **'start'** and each following bucket's upper bound is **'factor'** times the previous bucket's upper bound. The final +Inf bucket is not counted and not included in the returned slice. The returned slice is meant to be used for the Buckets field of HistogramOpts.
>
>The function panics if **'count'** is 0 or negative, if **'start'** is 0 or negative, or if **'factor'** is less than or equal 1.

Emphasis my own but the parameters are referenced in-between `'`

Reply:
Here's the link to the source code https://github.com/prometheus/client_golang/blob/master/prometheus/histogram.go#L88-L95